.travis.yml

```diff
@@ -28,10 +28,12 @@ before_script:
 
 script:
 # This step is required since we use a go module
+- cd ./v2
```

Comment:
Because working with the project is a little more tedious with the top level version directory present, I actually added a Makefile to the platform-services-go-sdk project.   Check it out to see if it might be useful for you as well:
https://github.com/IBM/platform-services-go-sdk/blob/master/Makefile
https://github.com/IBM/platform-services-go-sdk/blob/master/.travis.yml

With that sort of setup, I can run the following commands at the project root:
```
make all
make build
make unittest
make tidy
make lint
etc.
```

Reply:
@padamstx that's a good point. I will likely add Make and update the configuration in a future PR so that we can prioritize this release to resolve issues with moving to the latest major.